### DISCUSSION TITLE
Class not registered with AXIS Dewarping SDK CustomTransform sample

### DISCUSSION THREAD

**Discussion by kodonnell** [AxisCommunications/discussions#176] [Axis Windows SDKs] [answered]
Hi there, I'm helping a customer with their AXIS 3807, and we need to get off dewarped at higher FPS / resolution, which means client side dewarping. Unfortunately, I've downloaded the dewarping SDK but I can't get the samples to run. `CustomTransform` seems to be the one I want (i.e. convert 4x video recording to a dewarped video), but I get this error:

![image](https://user-images.githubusercontent.com/8204904/219574512-8b92af7f-bd0c-41c2-9c2b-1023f61f9cbe.png)

I'm not a C#/Windows dev, but I assumed it'd work out of box. The only thing I changed was when I opened it Visual Studio wanted to upgrade it to .NET Framework 4.8, and elevated privileges as I'm editing in `C:\Program files (x86)` etc. Otherwise, I'm on Windows 11 x64. It seems to be an issue with 32 vs 64 (and I note some other SDKs have both versions but this doesn't so e.g. [this](https://github.com/AxisCommunications/discussions/discussions/40#discussioncomment-3407466) answer won't work), and I tried manually registering the `AxisMediaViewerLib.dll` with `regsvr32` and the command succeeded, but didn't fix the above issue. 

Apologies if I missed something, or this is really silly - I looked for install instructions and couldn't find anything.

**Comment by anders-axis** [comment, marked as answer]
You probably need to install the AXIS Media Parser SDK.
